test/test_functional_tensor.py

```diff
@@ -226,11 +240,6 @@ def test_five_crop(self):
                                     (transforms.ToTensor()(cropped_pil_image[3]) * 255).to(torch.uint8)))
         self.assertTrue(torch.equal(cropped_tensor[4],
                                     (transforms.ToTensor()(cropped_pil_image[4]) * 255).to(torch.uint8)))
-        self.assertTrue(torch.equal(img_tensor, img_tensor_clone))
-        # scriptable function test
-        cropped_script = script_five_crop(img_tensor, [10, 10])
-        for cropped_script_img, cropped_tensor_img in zip(cropped_script, cropped_tensor):
-            self.assertTrue(torch.equal(cropped_script_img, cropped_tensor_img))
```

Comment:
Were those tests duplicate?

Reply:
Ops, this is accidentally removed due to https://github.com/pytorch/vision/pull/2568
I'll put it back